flake.nix

```diff
@@ -0,0 +1,36 @@
+{ 
+  description = "A lightweight, secure, and feature-rich Discord terminal client.";
+
+  inputs.nixpkgs.url = "nixpkgs/nixos-23.05";
+
+  outputs = { self, nixpkgs, ... }: let 
+
+    supportedSystems = [ 
+      "x86_64-linux" 
+      "aarch64-linux"
+      "x86_64-darwin"
+    ];
+
+    forSupportedSystems = f:
+      builtins.listToAttrs (builtins.map 
+        (system: 
+          { name = system; value = f system (import nixpkgs { inherit system; }); 
+        }) 
+        supportedSystems);
+    
+  in {
+    packages = forSupportedSystems (system: pkgs: 
+      let pkg = pkgs.buildGoModule {
```

Comment:
Needs to be `buildGo122Module` now as 0f5d835 bumped the required go version to >=1.22.1.